pitest/src/main/java/org/pitest/mutationtest/engine/gregor/ClassInfo.java

```diff
@@ -27,10 +25,6 @@ public boolean isEnum() {
     return this.superName.equals("java/lang/Enum");
   }
 
-  public boolean isGroovyClass() {
-    return FArray.contains(this.interfaces, isAGroovyClass());
-  }
-
   private static Predicate<String> isAGroovyClass() {
     return a -> a.startsWith("groovy/lang/")
         || a.startsWith("org/codehaus/groovy/runtime");
```

Comment:
I believe that private method could be removed as well.

Reply:
Btw, it seems to work, but I wonder why you check the "interfaces" here? 

I would expect checking if the "top super class" is GroovyObject, but there is just a superName, so it might be harder at that level :-/.

Reply:
This change is still in progress, will hopefully end up with all of ClassInfo being removed. It's a long time since I put this groovy filtering in place, looking at the tests that interface stuff might be required to detect closures (at least in the version of groovy that existed at the point I first wrote this).

Reply:
It might be even after you reported some issues with mutating Groovy code - yeaaaars ago :-). I missed the WIP status.